src/wp-admin/includes/class-wp-plugins-list-table.php

```diff
@@ -584,6 +582,25 @@ protected function get_views() {
 						$count
 					);
 					break;
+				default:
+					/**
+					 * Filter the status text of default switch case in the Plugins list table.
```

Comment:
Does this need to be capitalized?

```suggestion
					 * Filter the status text of default switch case in the plugins list table.
```